src/screens/Episode/helpers/dataChanger.ts

```diff
@@ -0,0 +1,21 @@
+import { Episode } from 'src/common/types'
+import {
+  DEFAULT_EPISODE_TRACK,
+  DEFAULT_EPISODE_IMAGE,
+  FormattedEpisodeData
+} from '../common'
+
+const dataChanger = (episode: Episode): FormattedEpisodeData => {
+  return Object.assign(
```

Comment:
We don't need `Object.assign` here, because you manually assign values. Also `Object.assign` doesn't work deeply